integration-test.ps1

```diff
@@ -60,11 +60,35 @@ Write-TaskName "Start Software Entitlement Server"
 $command = ".\sestest.ps1 server --connection $thumbprint --sign $thumbprint --encrypt $thumbprint --exit-after-request --log-file server.log --log-file-level debug"
 start-process powershell -argument $command
 
+Write-Output "Server started successfully."
+
 # ----------------------------------------------------------------------
 
-Write-TaskName "Wait 5s for server to be up"
+Write-TaskName "Wait for server to be up"
+
+function check-serverActive {
+    $address = [System.Net.IPAddress]('127.0.0.1')
+    $port = 4443
+    $client = New-Object System.Net.Sockets.TcpClient
+    try {
+        $client.Connect($address, $port);
+        return $true
+    }
+    catch {
+        return $false
+    }
+    finally {
+        $client.Dispose();
+    }
+}
+
+$ready = check-serverActive
+while (!$ready) {
```

Comment:
Why not just `while (!(check-serverActive)) …`

Reply:
Because that doesn't work. *It should*, but it doesn't. #blamingpowershell

Reply:
define doesn't work? 😊

Reply:
This loop:
```
while (!(check-serverActive))
{
    Write-Output "Waiting ..."
}
```
Never enters the loop and always moves on immediately. It also doesn't actually call the function - logging within the function doesn't appear.